delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -93,7 +98,7 @@ final class ResourcesRoutes(
                             .flatTap(index(project, _, mode))
                             .map(_.void)
                             .attemptNarrow[ResourceRejection]
-                            .rejectOn[InvalidSchemaRejection]
+                            .rejectWhen(rejectPredicateOnWrite)
```

Comment:
"reject when reject predicate on write" doesn't communicate what is happening here